components/policy-editor/org.wso2.carbon.policyeditor.ui/src/main/resources/org/wso2/carbon/policyeditor/ui/i18n/JSResources.properties

```diff
@@ -1,3 +1,21 @@
+#
+# Copyright (c) 2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
```

Comment:
Thanks, I'll double check with team about the exact year this file was created and if fail to find exact year replace with this 2015 which was the year I added license header.